pull-kubernetes-bazel-test fails:

```
W0227 02:17:57.895] 2018/02/27 02:17:57 error running compiler: exit status 1
I0227 02:17:57.996] pkg/volume/storageos/storageos_util_test.go:200:6: undefined: val
I0227 02:17:57.996] pkg/volume/storageos/storageos_util_test.go:201:66: undefined: val
W0227 02:17:58.719] INFO: Elapsed time: 190.906s, Critical Path: 33.26s
W0227 02:17:58.719] FAILED: Build did NOT complete successfully
```

--- a/pkg/volume/storageos/storageos_util_test.go
+++ b/pkg/volume/storageos/storageos_util_test.go
@@ -181,9 +181,9 @@ func TestCreateVolume(t *testing.T) {
 	if len(vol.Labels) == 0 {
 		t.Error("CreateVolume() Labels are empty")
 	} else {
+		var val string
+		var ok bool
 		for k, v := range labels {
-			var val string
-			var ok bool
 			if val, ok = vol.Labels[k]; !ok {
 				t.Errorf("CreateVolume() Label %s not set", k)
 			}
@@ -191,8 +191,6 @@ func TestCreateVolume(t *testing.T) {
 				t.Errorf("CreateVolume() returned unexpected Label value %s", val)
 			}
 		}
-		var val string
-		var ok bool
 		if val, ok = vol.Labels["labelfromapi"]; !ok {
 			t.Error("CreateVolume() Label from api not set")
 		}

